pkg/api/handlers/generic/images.go

```diff
@@ -315,3 +316,47 @@ func GetImages(w http.ResponseWriter, r *http.Request) {
 	}
 	utils.WriteResponse(w, http.StatusOK, summaries)
 }
+
+func LoadImages(w http.ResponseWriter, r *http.Request) {
+	// TODO this is basically wrong
+	decoder := r.Context().Value("decoder").(*schema.Decoder)
+	runtime := r.Context().Value("runtime").(*libpod.Runtime)
+
+	query := struct {
+		Changes map[string]string `json:"changes"`
+		Message string            `json:"message"`
+		Quiet   bool              `json:"quiet"`
+	}{
+		// This is where you can override the golang default value for one of fields
+	}
+
+	if err := decoder.Decode(&query, r.URL.Query()); err != nil {
+		utils.Error(w, "Something went wrong.", http.StatusBadRequest, errors.Wrapf(err, "Failed to parse parameters for %s", r.URL.String()))
```

Comment:
Maybe it's my fine Irish temper, but I hate seeing "Something went wrong" in an error message.  My first thought is "NO S**T!, now tell me what went wrong, don't waste time with info that doesn't help me solve my problem.".  I'd prefer just a plain simple "Error: "

But that might just be me.

Reply:
So true!

I fear we inherited that from Docker but I guess we can improve that for the libpod endpoints. @mheon @jwhonce @baude WDYT

Reply:
Please do improve on those messages.  In the swagger we use one model and that populates all the examples.  Brent and I planned to improve these in both trees over time.  The Cause and Message returns have also added to the "what do we want to say here" issue.